website/docs/components/table/table/partials/guidelines/guidelines.md

```diff
@@ -159,45 +152,44 @@ When using striping in a Table, the first tinted row will be the second row of t
 The row placement property is only relevant in Figma and doesn’t exist as a property in code.
 !!!
 
-Row placement determines the visual styling based on where the row is placed relative to other rows within the Table. Only cells with a column placement that is either `start` or `end` utilize the row placement property; column position `middle` does not utilize this property.
+Row placement determines the visual styling based on where the row is located. Only cells with a column placement that is `start` or `end` utilize the row placement property; column position `middle` does not utilize this property.
 
 ![Table row placement example](/assets/components/table/table-row-placement.png)
 
 ## Headers
 
-- Headers should be clear, concise, and straightforward.
-- The text should be descriptive of the type of content contained within the cell, e.g. Created date, Email, Project name.
-- Headers should use sentence-case capitalization, not all-caps.
+- Labels in headers should be concise and straightforward.
+- The label should clearly indicate what type of content is contained within the cell (e.g. Created date, Email, Project name).
+- Labels should always use sentence-case capitalization.
 
 ## Cells
 
-Cells are the core organizational container within a Table. In order for the user to be able to sort and read the table easily, each cell in a table should contain one piece of data. Having more than one piece of information inside a cell also makes it harder for users to navigate the relationships between headers and content. 
+In order for the user to sort and read the table easily, each cell in a table should contain one piece of data. Having more than one piece of information inside a cell makes it harder for users to navigate the relationships between headers and content. 
 
 ### Density
 
-- We recommend using `medium` cell density by default.
+- Use `medium` density by default for balanced readability and display.
+- Choose `short` density for text-heavy tables to fit more rows on a page but can make tables harder to read and scan, so use it thoughtfully.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
```

Comment:
```suggestion
- By default, use the `medium` density for balance and readability.
- To fit more rows on a page, use the `short` density. Use this only for text-heavy tables, as it can make them harder to scan.
- For a smaller dataset, e.g., basic user data, consider using the `tall` density to provide the content with more breathing room.
```